sim/dex-data.js

```diff
@@ -916,7 +916,7 @@ class Move extends Effect {
 		 * Will this move always or never be a critical hit?
 		 * @type {boolean | undefined}
 		 */
-		this.willCrit = this.willCrit || undefined;
+		this.willCrit;
```

Comment:
```suggestion
		this.willCrit = this.willCrit || undefined;
```

By the way, I already fixed this server-side, so this should be reverted to prevent a conflict. I still appreciate the test, though!